src/thinker.rs

```diff
@@ -23,27 +22,41 @@ use crate::{
 };
 
 /**
-Wrapper for Actor entities. In terms of Scorers, Thinkers, and Actions, this is the [`Entity`] actually _performing_ the action, rather than the entity a Scorer/Thinker/Action is attached to. Generally, you will use this entity when writing Queries for Action and Scorer systems.
+ * Wrapper for Actor entities. In terms of Scorers, Thinkers, and Actions, this
+ * is the [`Entity`] actually _performing_ the action, rather than the entity a
+ * Scorer/Thinker/Action is attached to. Generally, you will use this entity
```

Comment:
This is one of the more confusing lines of the docs, I think.

In my case, the entity performing the action IS the entity with a ThinkerBuilder component.

A new user doesn't understand the difference between Action and ActionBuilder, (same for scorer/thinker), so this line just confuses them more, I think

Reply:
yes, I think your third paragraph is the issue here, really.

One of the things I've struggled with most w/ Big Brain ergonomics is how to have the Action/ActionBuilder (etc) distinction without overwhelming new users, but this might be because I've tried to hard to sweep it under the rug in "basic" cases...